lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -57,7 +58,7 @@ def vetext_incoming_forwarder_lambda_handler(event: dict, context: any):
         logger.exception(e)
         push_to_dead_letter_sqs(event, "vetext_incoming_forwarder_lambda_handler")
 
-        return create_twilio_response()
+        return create_twilio_response(500)
```

Comment:
`push_to_dead_letter_sqs()` is called in 5 or so places. Need to ensure the response to Twilio is 500 any time we push to the dead letter queue.

Reply:
@ianperera -  **Note** process_body_from_sqs_invocation and push_to_retry_sqs methods that call the push to dead letter method dont need to return 500.  I think it's only line 42 that needs it added.